browser.py

```diff
@@ -49,7 +49,7 @@ def __load_finished(self, view, frame):
         source = frame.get_data_source()
         data = source.get_data()
         if data is not None:
-            self.emit("load-finished", data.str)
+            self.emit("load_changed", data.str)
```

Comment:
This is wrong.  The class does not have a load-changed signal; line 34 still has it as load-finished.  Do you also need to change the name of the signal here and in the caller?